cmake/GenerateRequirementsTable.py

```diff
@@ -66,10 +73,14 @@ class Requirement:
 
     descr = None
     tix = None
+    tname = None
 
-    def __init__(self, ID, testFile, rawReqBlock):
+    def __init__(self, ID, testName, testFile, rawReqBlock):
         self.ID = ID
-        self.tfile = testFile
+        self.tfile = testFile[len(args.path)+1:]
```

Comment:
This is going to be problematic if a user specifies their path as:

--path /path/to/file/

instead of

--path /path/to/file

For me, it was cutting off the first character in my path and it took me a while to track down why.  The better solution here would be to use os.path.relpath to get the location you want.  For example:

```
self.title = os.path.relpath(testFile, args.path)
```

Reply:
Addressed this, please check.